pyteal/ast/abi/address_test.py

```diff
@@ -1,5 +1,12 @@
+import pytest
 import pyteal as pt
 from pyteal import abi
+from pyteal.ast.abi.address import ADDRESS_LENGTH_BYTES
+
+from pyteal.ast.global_ import Global, GlobalField
```

Comment:
`Global` and `GlobalField` should be imported in the `from pyteal` line, since they are accessible from the top-level package

`abi.ADDRESS_LENGTH_BYTES` should be used, not `from pyteal.ast.abi.address import ADDRESS_LENGTH_BYTES`, so that we're testing what's exported

Recently added this justification to the repo here: https://github.com/algorand/pyteal/blob/master/CONTRIBUTING.md#in-test-code